source/versioned-api/tests/test-commands-strict-mode.yml

```
@@ -6,6 +6,8 @@ runOnRequirements:
   - minServerVersion: "4.9"
     serverParameters:
       enableTestCommands: true
+    # serverless does not support the testVersion2 command
```

Nit: this sounds as if `testVersion2` is supported in some cases. In this case serverless gives a different error:
```
{
        "ok" : 0,
        "errmsg" : "command not found",
        "code" : 59,
        "codeName" : "CommandNotFound"
}
```

Perhaps reword to this?

```suggestion
    # serverless gives a different error for unrecognized testVersion2 command
```